packages/react-components/react-skeleton/src/components/SkeletonCircle/useSkeletonCircle.ts

```diff
@@ -15,17 +16,15 @@ export const useSkeletonCircle_unstable = (
   props: SkeletonCircleProps,
   ref: React.Ref<HTMLElement>,
 ): SkeletonCircleState => {
+  const { radius = '24px' } = props;
+
+  const root = getNativeElementProps('div', { ref });
```

Comment:
you're also missing the props spread here

```suggestion
  const root = getNativeElementProps('div', { 
    ref,
    ...props
  });
```